Products/CMFPlone/controlpanel/browser/maintenance.pt

```diff
@@ -105,6 +105,10 @@
                             tal:condition="form_name"
                             tal:content="form_name">Form name</legend>
 
+                    <p>
+                        <strong i18n:translate="text_zope_database_name">Database name:</strong> <span tal:replace="view/dbName" />
```

Comment:
Ah, I see. You want an ``i18n:name`` on it. I misunderstood that as an ``i18n:attribute``. But still: shouldn't the next paragraph changed instead? AFAIK you cannot translate text with html tags inside. Or how would you preserve the ``<strong>`` tag, when you apply the ``i18n:translate`` attribute at the surrounding paragraph, like below?